sync15-adapter/src/key_bundle.rs

```diff
@@ -86,8 +86,9 @@ impl KeyBundle {
         Ok(out)
     }
 
+    /// Important! Don't compare against this directly! use `verify_hmac` or `verify_hmac_string`!
```

Comment:
Do you think the HMAC could be made its own type that doesn't implement `Eq`, or implements it as a constant time comparison under the hood...or is that not worth the trouble?

Reply:
We don't compare HMACs in enough places for this to make sense to me.